### PR TITLE
simple-product-page-layout-1

### PR DESCRIPTION
**Pull request by bipin7yadav**

### CODE REVIEW

File: product-page-layout-1/layout-one.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    --whitesmoke-color:whitesmoke;
+    --alice-blue-color:aliceblue;
+}
+
+.container{
+    width: 100%;
+    background-color: var(--white-color);
+    margin: .9rem;
+}
+#header{
```
Comment (msdeshmukh009) [inline product-page-layout-1/layout-one.css:16]: Maybe avoid Id selectors for styling.

File: product-page-layout-1/layout-one.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    padding: .4rem;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin-top: .5rem;
+    margin-right: .5rem;
+}
+
+
+img{
```
Comment (msdeshmukh009) [inline product-page-layout-1/layout-one.css:75]: Not a good idea to style HTML elements directly.

File: product-page-layout-1/layout-one.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.container{
+    width: 100%;
+    background-color: var(--white-color);
+    margin: .9rem;
+}
+#header{
+    display: flex;
+    justify-content: space-around;
+    background-color: var(--black-color);
+    margin: .3rem;
```
Comment (msdeshmukh009) [inline product-page-layout-1/layout-one.css:20]: Try to maintain a jump of 8px or 0.5rem in margin and padding whenever you can. As that will make the UI more consistent.

**Review by manojkumar8400 — APPROVED** [review]
good to go.

**Pull request merged**